docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - -c
       - "npm start"
   mojaloop-testing-toolkit-ui:
-    image: mojaloop/ml-testing-toolkit-ui:v11.0.2
+    image: mojaloop/ml-testing-toolkit-ui:v11.0.4
```

Comment:
I think we should change the version of docker image only after releasing the version for UI. For now let's revert this back